src/configuration.cc

```diff
@@ -31,19 +31,24 @@ constexpr const char kDefaultProjectId[] = "";
 constexpr const char kDefaultCredentialsFile[] = "";
 constexpr const int kMetadataApiDefaultNumThreads = 3;
 constexpr const int kMetadataApiDefaultPort = 8000;
+constexpr const char kMetadataApiDefaultResourceTypeSeparator[] = ".";
 constexpr const int kMetadataReporterDefaultIntervalSeconds = 60;
 constexpr const char kMetadataIngestionDefaultEndpointFormat[] =
     "https://stackdriver.googleapis.com/v1beta2/projects/{{project_id}}"
     "/resourceMetadata:batchUpdate";
 constexpr const int kMetadataIngestionDefaultRequestSizeLimitBytes =
     8*1024*1024;
+constexpr const char kMetadataIngestionDefaultRawContentVersion[] = "0.1";
+constexpr const int kInstanceUpdaterDefaultIntervalSeconds = 60*60;
+constexpr const char kDefaultInstanceResourceType[] = "";
```

Comment:
Should this be blank?

Reply:
Blank means "get it from the environment".

Reply:
Just added a comment to that effect.